apps/examples/testcase/le_tc/kernel/tc_mqueue.c

```diff
@@ -866,5 +945,8 @@ int mqueue_main(void)
 	tc_mqueue_mq_timedsend_timedreceive_failurechecks();
 	tc_mqueue_mq_unlink();
 
+	tc_mqueue_mq_getattr();
```

Comment:
If possible, arrange in the alphabetical order.

Reply:
Is it really necessary.
I think it is fine like this only.